prometheus/src/main/scala/filodb/prometheus/ast/Vectors.scala

```diff
@@ -12,6 +12,11 @@ object Vectors {
   val BucketFilterLabel = "_bucket_"
 }
 
+
+object WindowConstants {
+  val staleDataLookbackSeconds = 5 * 60 // 5 minutes
```

Comment:
Yes - Fully agree that it should. I'd like for that to be in a different PR since getting config down to this place correctly requires some tricks. This PR just moves this config from one place to another.